site/src/testHelpers/entities.ts

```diff
@@ -814,15 +814,29 @@ export const MockCancellationMessage = {
   message: "Job successfully canceled",
 }
 
-// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
Previously, ESLint was unable because we weren't defining the return type.

I decided to break it out into three type aliases: input, output and the function. Makes it a bit more readable too 👍🏼